### DISCUSSION TITLE
Request for Licensing Info: Logo and Front-End Component Modification

### DISCUSSION THREAD

**Discussion by martin0258** [langgenius/dify#3291] [Help]
Hi Dify Team,

After trying out Dify internally, currently I'm considering to customize Dify for one of our clients in an offline deployment scenario, and we may modify Dify's logo to our client's branding. Therefore, I'm reaching out to inquire about obtaining a license for modifying the Dify logo and front-end components, according to your [LICENSE](https://github.com/langgenius/dify/blob/main/LICENSE).

Could you kindly provide the following information:
1. Licensing fee for the aforementioned modifications.
2. Terms and conditions for the license.
3. Application process for obtaining the license.

I have sent an email to business@dify.ai on 4/4 but no response. Look forward to your response!

**Comment by wojiaowenjing** [comment]
您解决了吗？我们企业也希望能修改这些

**Comment by hly9581** [comment]
请问您解决了吗？我们也想修改这些
